### DISCUSSION TITLE
Get code from codesandbox

### DISCUSSION THREAD

**Discussion by dreamniker** [codesandbox/codesandbox-client#1028] [General] [answered]
# ❔ Question

Hey. Is it possible to embed the codesandbox-client into my website and save the code, that I write inside the codesandbox to my server? Now, I found only one solution:
Open the code in the codesandbox through the Define API> edit -> download zip archive -> Download it to my server and then open it using Define API
Thanks :)

**Comment by lbogdan** [comment, marked as answer]
Hey @dreamniker 

There is an API endpoint that returns the contents of a sandbox: https://codesandbox.io/api/v1/sandboxes/new (replace `new` with your sandbox id).

If that answers your question, please help us keep the issue count low by closing this! 🙂
